README.md

```diff
@@ -18,13 +18,26 @@ limitations under the License.
 
 This repository contains hooks for use with [pre-commit](pre-commit.com).
 
+## Table of contents
```

Comment:
Duplicate heading?

Reply:
Yep, need to remove it. Ironically, this is part of what I'm doing -- having this header generated instead of manually putting it everywhere. Doesn't stop me from making the same mistake, clearly. :)